common/src/main/java/org/valkyrienskies/mod/mixin/client/MixinMinecraft.java

```diff
@@ -159,4 +159,14 @@ public void deleteShipObjectWorldClient() {
         shipObjectWorldCopy.destroyWorld();
         shipObjectWorld = null;
     }
+
+    @Inject(
+        method = "clearLevel",
+        at = @At("TAIL")
```

Comment:
Should this be @At("RETURN") instead?

Reply:
Method doesn't return anything so it basically synonymous